app/src/main/java/org/openobservatory/ooniprobe/common/OrchestraTask.java

```diff
@@ -36,10 +37,15 @@ public static void sync(Application app) {
 					app.getFilesDir() + "/orchestration_secret.json") ;
 			//TODO ORCHESTRATE
 			//client.setAvailableBandwidth(String value);
-			//what happens when token is nil? should register anyway with empry string
-			client.setCABundlePath(app.getCacheDir() + "/" + Application.CA_BUNDLE);
-			client.setGeoIPCountryPath(app.getCacheDir() + "/" + Application.COUNTRY_MMDB);
-			client.setGeoIPASNPath(app.getCacheDir() + "/" + Application.ASN_MMDB);
+			//what happens when token is nil? should register anyway with empty string
```

Comment:
What token?

Reply:
it's an ancient comment, not to be addressed now